Misc/NEWS.d/next/Core_and_Builtins/2023-04-29-23-15-38.gh-issue-103997.BS3uVt.rst

```diff
@@ -0,0 +1,4 @@
+String arguments passed to "-c" are now automatically dedented as if by
```

Comment:
This is a new feature so it requires an additional What's New entry.

Reply:
I don't have time to look into this. Can you check it? @Erotemic

Reply:
It's now done, thanks @methane